tests/relational-lenses/serial.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS t1;
+
+CREATE TABLE t1 (
+    id1 SERIAL NOT NULL,
```

Comment:
I think SERIAL is potsgres-specific while the sqlite3 version below is generic SQL.  So I'd be inclined to name this file "serial.pgsql" and rename the sqlite3-specific one to use the .sql suffix.

It is too bad that they didn't standardize this, since in my experience every nontrivial database application uses serial/auto-increment etc.

Reply:
I have renamed it to `serial.pgsql`, but I don't think the sqlite3 one deservers to receive the generic `.sql` either. I find Sqlite3's behaviour very implicit, as setting a column type to `INTEGER PRIMARY KEY` converts it into a column alias for rowid. Having something slightly different like `INT PRIMARY KEY` already seemingly breaks this behaviour. I personally find postgres' behaviour of introducing a new datatype with a different behaviour better, but agree it should probably not be the default file.